### ISSUE TITLE
CST - Heading order is invalid for status tab past events heading

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#107886]
## Issue
The H3 that shows under the Status tab component is out of order. It goes H1 -> H3 -> H2 when the past events are shown under the Status tab. 

New Cypress Axe rules for headings are coming and will be put into effect April 30. [Slack thread](https://dsva.slack.com/archives/C03R5SBELQM/p1742920275472879) The link to the Axe report expired, but I noticed this one while running an Axe Devtools scan.

Page: Claim details pages that show the Status tab with past events. TBD on which ones these are, but probably a design system component to be fixed.

<img width="1439" alt="Image" src="https://github.com/user-attachments/assets/ba80a5f6-004a-4a1f-9745-ae9d680fb6be" />

**Comment by bacitracin** [comment]
Covered in 108007, closing as dupe
